.pipelines/templates/publish-helm-chart.yaml

```diff
@@ -4,15 +4,15 @@ steps:
       az acr login --name $(registry.name)
     displayName: Login to acr
   - bash: |
-      cp $(image.dir)/$(chart.name)-$(image.tag).tgz .
+      cp $(image.dir)/$(chart.name)-$(image.tag)-${UNIQUE_TAG}.tgz .
```

Comment:
can you use CHECKOUT_TAG here and throughout the file? Is there a way to define it once somewhere for all these templates?

Reply:
Yes, you can define it in the variables section. I can use checkout.tag instead of image.tag